test/fuzz/lua_fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# External dependencies
+include(ProtobufMutator)
+include(protobuf)
```

Comment:
These includes will produce conflict with two or more fuzzers that depend on LPM and LP. To avoid that, these includes should be placed in `test/fuzz/CMakeLists.txt` before `add_subdirectory(...)` for all fuzzers.

Reply:
Moved to test/fuzz/CMakeLists.txt.